pkg/storage/utils/templates/templates.go

```diff
@@ -95,6 +106,24 @@ func WithSpacePropertiesAndUser(u *userpb.User, spaceType string, spaceName stri
 	return b.String()
 }
 
+// WithResourceInfo renders template stings with ResourceInfo variables
+func WithResourceInfo(i *providerv1beta1.ResourceInfo, tpl string) string {
+	tpl = clean(tpl)
+	data := newResourceData(i)
+	// compile given template tpl
+	t, err := template.New("tpl").Funcs(sprig.TxtFuncMap()).Parse(tpl)
+	if err != nil {
+		err := errors.Wrap(err, fmt.Sprintf("error parsing template: fileinfoandresourceid_template:%+v tpl:%s", data, tpl))
+		panic(err)
```

Comment:
Please don't panic. Return the error or ignore it.

Reply:
ok, that means we should also refactor the other templates?
IMO it was designed that way that you cannot start the serivice with a bad template.

Reply:
It would still not start if we return the error, right?

Apparently I didn't review the first part. If there are already panics we can keep these for now and refactor later.